python/pyspark/ml/regression.py

```diff
@@ -743,6 +743,18 @@ def treeWeights(self):
         """Return the weights for each tree"""
         return list(self._call_java("javaTreeWeights"))
 
+    @property
```

Comment:
I've made a JIRA to follow up with this post 2.0 release ( https://issues.apache.org/jira/browse/SPARK-15161)